docs/HelmChart.md

```diff
@@ -0,0 +1,317 @@
+# Deploying GUAC with Helm Chart
+
+This tutorial introduces the full GUAC components deployment and how to deploy
+it with the [GUAC Helm Chart](https://github.com/kusaridev/helm-charts/tree/main/charts/guac).
```

Comment:
We can probably prefix this with "Kusari's third party GUAC Helm Chart". Since technically ownership is not part of this repo, it acts as a way to link and shout out to Kusari's work here as well. thoughts? Or is the idea to donate the helm chart and then make an announcement around that? @sunnyyip @mlieberman85 @trmiller

Reply:
added kusari to the helm chart link